solutions/dedicated/variables.tf

```diff
@@ -0,0 +1,67 @@
+variable "ibmcloud_api_key" {
+  description = "The IBM Cloud API key to deploy IAM-enabled resources."
+  type        = string
+  sensitive   = true
+}
+
+variable "region" {
+  type        = string
+  description = "The IBM Cloud region where the instance of IBM Cloudant is provisioned."
+  default     = "us-south"
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group. If false, specify the name for the new resource group in the `resource_group_name` input."
```

Comment:
Remove this line: "If false, specify the name for the new resource group in the `resource_group_name` input."

`resource_group_name` is required regardless of whether this variable is true or false